README.md

```diff
@@ -142,11 +142,11 @@ Related objects can also be produced using the `producers.relationship` function
 
 ```python
 project = projectors.combine(
-    projectors.wrap_producer("name", producers.attr("name")),
+    projectors.producer_to_projector("name", producers.attr("name")),
```

Comment:
I find this extremely verbose but if its internal only I guess its better to be explicitly very explicit and explicit and verbose and extra verbosely explicit